### PR TITLE
Coalesce render flavors

### PR DESCRIPTION
**Pull request by michaelkirk**
Just a little widgetry cleanup

### CODE REVIEW

File: widgetry/src/text.rs [hunk 1]
```diff
@@ -271,18 +270,11 @@ impl Text {
         self.render(assets).get_dims()
     }
 
-    // TODO: Combine all render_foo methods to be render(into<Assets>)?
-    pub fn render(self, assets: &Assets) -> GeomBatch {
+    pub fn render<'a, A: AsRef<Assets>>(self, assets: &A) -> GeomBatch {
```
Comment (michaelkirk) [inline widgetry/src/text.rs:273]: Thrust of the PR is here

File: widgetry/src/assets.rs [hunk 2]
```diff
@@ -1,3 +1,5 @@
+use crate::drawing::GfxCtx;
```
Comment (dabreegster) [inline widgetry/src/assets.rs:1]: Nit: Since these're exported from the crate, could just do these on the `use crate::{text, GeomBatch}` line

**Review by dabreegster — CHANGES_REQUESTED** [review]
Nice cleanup, thanks! Just the one import nit, then we can merge

**Comment by michaelkirk** [comment]
Fixed, PTAL @dabreegster

**Review by dabreegster — APPROVED** [review]

**Pull request merged**